Gems/Multiplayer/Code/Source/NetworkEntity/NetworkSpawnableLibrary.cpp

```diff
@@ -42,7 +41,8 @@ namespace Multiplayer
 
         auto enumerateCallback = [this](const AZ::Data::AssetId id, const AZ::Data::AssetInfo& info)
         {
-            if (info.m_assetType == AZ::AzTypeInfo<AzFramework::Spawnable>::Uuid())
+            if (info.m_assetType == AZ::AzTypeInfo<AzFramework::Spawnable>::Uuid() &&
+                info.m_relativePath.ends_with(".network.spawnable"))
```

Comment:
Maybe it's time to create a named constant for ".network.spawnable" and replace it in the other 2 places too?

Reply:
good call out, on it!

Reply:
fixed!